mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -427,16 +430,19 @@ def validator_default(utterance):
         on_fail_fn = on_fail if callable(on_fail) else on_fail_default
         validator = validator or validator_default
 
+        # Setup the converse handler
```

Comment:
is this included accidentally? all changes in this whole file doesn't seem XDG related

Reply:
I believe it's to fix some issues with VK that this PR uncovered? @forslund made the change so knows more about it.

Reply:
That should probably be reverted, it's from one of the many attempts at figuring out the issue with the failing timer skill. It didn't fix it completely so it needs to be looked at again. I do believe the timer skill does something really funky, looking at the logs it throws a couple of random errors (even for successful test runs)